### PR TITLE
Mark many strings for translation

### PR DESCRIPTION
**Pull request by aeolianeth**
Mark the main app strings for translation.

This excludes Project Creation strings, as the copy is currently being worked on and subject to change.

### CODE REVIEW

File: src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -60,19 +61,19 @@ export default function Navbar() {
...
               ?.scrollIntoView({ behavior: 'smooth' })
           }, 0)
         })}
-        {menuItem('Docs', 'https://docs.juicebox.money')}
-        {menuItem('Blog', 'https://blog.juicebox.money')}
+        {menuItem(t`Docs`, 'https://docs.juicebox.money')}
+        {menuItem(t`Blog`, 'https://blog.juicebox.money')}
         {menuItem('Discord', 'https://discord.gg/6jXrJSyDFf')}
-        {menuItem('Workspace', 'https://juicebox.notion.site')}
+        {menuItem(t`Workspace`, 'https://juicebox.notion.site')}
```
Comment (aeolianeth) [inline src/components/Navbar/index.tsx:76]: Thoughts on removing this from the main nav?

**Comment by aeolianeth** [comment]
Hey @peripheralist, I've gone through and marked a bunch of strings for translation. This PR doesn't include any translations, though. Hoping to get folks translating directly to see if the GitHub translation workflow is adequate 😉

**Comment by cunningproof** [comment]
Reviewed this to get up to speed on the i18n progress; looks good to me.

UI element to manually set lang coming once translations are created then ?
`detect(fromUrl('lang'), fromStorage('lang'), fromNavigator()) ??`

**Comment by aeolianeth** [comment]
@cunningproof yep exactly!

We should eventually add a basic language selecting mechanism from the UI, though.

File: src/components/Landing/Footer.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ export default function Footer() {
     >
       <div style={{ display: 'inline-flex', justifyContent: 'center' }}>
         {link('Discord', 'https://discord.gg/6jXrJSyDFf')}
-        {link('Github', 'https://github.com/jbx-protocol/juicehouse')}
+        {link('GitHub', 'https://github.com/jbx-protocol/juice-interface')}
```
Comment (peripheralist) [inline src/components/Landing/Footer.tsx:32]: 👍

File: src/components/Navbar/index.tsx (hunk 1 above)
Comment (peripheralist) [reply src/components/Navbar/index.tsx:76]: This was included a while back at the request of the folks organizing Notion, I think it helps with pointing newcomers there. Let's leave it for now, but throw out a question in the Discord if you're keen on removing it

**Review by peripheralist — APPROVED** [review]
awesome

**Comment by cunningproof** [comment]
> We should eventually add a basic language selecting mechanism from the UI, though.

yeah, that's what I meant. After some translations are added - so when I hit it from a vpn I'm not lost :)

**Comment by aeolianeth** [comment]
@cunningproof ah my bad! I misread your comment.

Issue created for this: https://github.com/jbx-protocol/juice-interface/issues/276. Drop any thoughts you have in there!

File: src/components/Navbar/index.tsx (hunk 1 above)
Comment (aeolianeth) [reply src/components/Navbar/index.tsx:76]: Thanks for context, no worries, I don't feel strongly!

**Pull request merged**